### PR TITLE
Add disclaimer dialog

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

Add disclaimer dialog

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

스크린샷 2023-06-01 오후 10 21 25

### CODE REVIEW

File: src/components/common/ModalDisclaimer.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        <div class="text--lg">{{ $t('disclaimer.chapter3') }}</div>
+        <div class="text--md">
+          {{ $t('disclaimer.para2') }}
+        </div>
+        <div class="text--md">
+          {{ $t('disclaimer.closeWord') }}
+        </div>
+      </div>
+
+      <div class="container--buttons">
```
Comment (bobo-k2) [inline src/components/common/ModalDisclaimer.vue:34]: I see dark border around buttons in light theme
image

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: src/components/common/ModalDisclaimer.vue [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  </astar-default-modal>
+</template>
+
+<script lang="ts">
+import { truncate, wait } from '@astar-network/astar-sdk-core';
+import { fadeDuration } from '@astar-network/astar-ui';
+import { defineComponent, ref } from 'vue';
+import { LOCAL_STORAGE } from 'src/config/localStorage';
+
+export default defineComponent({
```
Comment (bobo-k2) [inline src/components/common/ModalDisclaimer.vue:56]: Add test(s) for the component

**Comment by Kahonnohak** [comment]
Could we add a delay somehow? As it's quite "not inviting". It would be better to be able to see Portal and maybe a trigger to show the modal. Maybe when user scroll or click? What would you think?

**Comment by sirius651** [comment]
> Could we add a delay somehow? As it's quite "not inviting". It would be better to be able to see Portal and maybe a trigger to show the modal. Maybe when user scroll or click? What would you think?

Added delay about 2sec. please check and let me know if suitable

**Comment by Kahonnohak** [comment]
Looks good! This won't be added to the next cycle since the disclaimer itself is being reviewed right now. I will update once its approved!

**Review by impelcrypto — CHANGES_REQUESTED** [review]

**Review by impelcrypto — APPROVED** [review]

**Review by bobo-k2 — APPROVED** [review]

**Comment by Kahonnohak** [comment]
Hi, @bobo-k2 Please help when you have a spare time.

We need to add two links that we were waiting for.

**Terms of Service** (font weight 600 and Astar Blue)
https://docs.google.com/document/d/1gxM0PEzFq7nW5VB11pMcDUaaKxfMz3BjTDtmEem_oo4/edit#heading=h.hqbzr87xmci3

**Privacy Policy** (font weight 600 and Astar Blue)
https://docs.google.com/document/d/1jEbhRfh292TahRMRdeN4z-8MYNU27dCS_vVopV6xQgk/edit#heading=h.cey5hqakqpbb
Screenshot 2023-07-14 at 12 05 53

**Review by bobo-k2 — CHANGES_REQUESTED** [review]
Add links

**Comment by Kahonnohak** [comment]
We got proper links now! @gluneau could you please replace them?

Terms of Use
https://astar.network/term-of-use

Privacy policy
https://astar.network/privacy-policy

**Review by Kahonnohak — APPROVED** [review]
Thanks! Good to go!

**Pull request merged**